### DISCUSSION TITLE
Supported devices - Will the E1 outdoor ever be supported

### DISCUSSION THREAD

**Discussion by turboboost** [fwestenberg/reolink_dev#424] [Q&A]
..or are the whole E1 family out of scope?
E1 outdoor works  apart from the PTZ stuff that I need.. or at least would like..

**Comment by starschild86** [comment]
interested as well, after some research i would like to buy a e1 outdoor but not if it's not well integrated within HA.

at the moment E1 outdoor works well with just ptz excluded? for it you need to open the mobile app?

**Comment by kshannon1** [comment]
E1 support would be nice.  Thanks.

**Comment by conma293** [comment]
any update on this? From what I can see E1 can be seen in the dashboard? thats all I want.... a siren switch entity would be nice but... deciding between a Reolink and a Wyzev3...
